torchtune/utils/_import_guard.py

```diff
@@ -10,3 +10,12 @@
 _SUPPORTS_FLEX_ATTENTION = (
     torch.cuda.is_available() and torch.cuda.get_device_capability() >= (7, 5)
 )
+
```

Comment:
nit: Ed says we shouldn't do this anymore :/


https://fb.workplace.com/groups/pytorch.dev/permalink/1677770972801376/